cmd/gaiad/testnet.go

```diff
@@ -33,6 +24,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
 	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/spf13/cobra"
```

Comment:
undo. This is not the correct import structure. Local imports should be grouped by:

- stdlib
- github.com/cosmos/gaia
- everything else

Reply:
If you use VSCode, have the following in your workspace settings:

```json
  "gopls": {
    "local": "github.com/cosmos/gaia",
  },
```

Reply:
Please use `goimports` @jgimeno

Reply:
I've run `make format`. That should do it. Hence resolving.